.eslintrc.js

```diff
@@ -102,5 +102,88 @@ module.exports = {
         'valid-typeof': 'off',
       },
     },
-  ],
-}
+    {
+      files: [
+        "lib/index-bundle.ts",
+        "lib/index.ts",
+        "lib/network/gephiParser.ts",
+        "test/NodesHandler.test.ts",
+        "test/dot-parser/dot-parser.test.ts",
+        "test/edges/bezier-edge-dynamic.test.ts",
+        "test/edges/bezier-edge-static.test.ts",
+        "test/edges/cubic-bezier-edge.test.ts",
+        "test/edges/edge-base.test.ts",
+        "test/edges/end-points.test.ts",
+        "test/edges/helpers.ts",
+        "test/edges/straight-edge.test.ts",
+        "test/gephi-parser.test.ts",
+        "test/helpers/index.ts"
+      ],
+      rules: {
+        "prettier/prettier": [
+          "error",
+          {
+            endOfLine: "lf",
```

Comment:
@Thomaash I thought we decided to use the default prettier config?

Reply:
This is just for existing files. The default config will be enforced in new files.

Reply:
I think we should leave th old *.js files alone and just format the new *.ts files.

Reply:
Eh, this is exactly what this config does.